curriculum/challenges/english/25-front-end-development/quiz-backend-javascript/66f1b11026394bcdb7c42473.md

```diff
@@ -17,375 +17,374 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which command initializes a new package.json file in npm?
 
 #### --distractors--
 
-Placeholder distractor 1
+npm create
 
 ---
 
-Placeholder distractor 2
+npm add
 
 ---
 
-Placeholder distractor 3
+npm start
 
 #### --answer--
 
-Placeholder answer
+npm init
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary purpose of `npm`?
 
 #### --distractors--
 
-Placeholder distractor 1
+To run a web server
 
 ---
 
-Placeholder distractor 2
+To bundle code for production
 
 ---
 
-Placeholder distractor 3
+To compile JavaScript
 
 #### --answer--
 
-Placeholder answer
+To manage packages for Node.js
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which JavaScript engine is used by Node.js?
 
 #### --distractors--
 
-Placeholder distractor 1
+JavaScriptCore
 
 ---
 
-Placeholder distractor 2
+SpiderMonkey
 
 ---
 
-Placeholder distractor 3
+Chromium
```

Comment:
```suggestion
Chakra
```
I think Chakra, or some other actual engine, makes more sense here. Unless it is intentional that we use something that isn't an engine.

Reply:
Yeah, I got your point. I will make the required changes